css/style.css

```diff
@@ -8738,4 +8738,73 @@ dt::last-of-type {
 .faqb {
  font-weight: bold;
  padding: 2em;
-}
\ No newline at end of file
+}
+
+
+
+/* -------------------------------------------------- 
+	Code of conduct
+	/* -------------------------------------------------- */
+
+/* Variables */
+/* Mixins */
+.container1 {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-wrap: wrap;
+  height: 47vh;
+}
+.container1 .box {
+  width: 220px;
+  height: 200px;
+  border-radius: 10px;
+  background-color: white;
+  margin: 20px;
+  transition: 0.3s all ease-in-out;
+  position: relative;
+  font-family: "Dosis", sans-serif;
+  box-shadow: -10px 25px 50px rgba(0, 0, 0, 0.3);
```

Comment:
try decreasing the box-shadow a bit.

Reply:
And maybe make the on hover turn the borders darker instead of the current version...